fuse_constraints/include/fuse_constraints/normal_delta_pose_3d_cost_functor.h

```diff
@@ -88,7 +93,41 @@ class NormalDeltaPose3DCostFunctor
     const T* const orientation1,
     const T* const position2,
     const T* const orientation2,
-    T* residual) const;
+    T* residual) const {
```

Comment:
/tmp/ws/src/fuse/fuse_constraints/include/fuse_constraints/normal_delta_pose_3d_cost_functor.h:96:  when starting a new scope, { should be on a line by itself  [whitespace/braces] [4]

Reply:
Also above it looks like the style has changed.

Unless you @svwilliams have a `.clang-format` file we could use, I think you @jakemclaughlin6 will have to fix the style manually here and in the rest of the file. Just preserved the style from the original code when you moved it here.